torch/csrc/generic/methods/TensorRandom.cwrap

```diff
@@ -228,6 +228,15 @@
           default: THPDefaultGenerator->cdata
           kwarg_only: True
         - THTensor* alpha
+    - cname: standard_gamma_alpha_with_grad
```

Comment:
It's bette to expose this as an internal method instead of a new overload (think `torch._C._standard_gamma_alpha_with_grad`)

Reply:
Could you point me to an example internal method whose plumbing I can copy? I'm a little lost here.